eo-maven-plugin/src/main/resources/org/eolang/maven/transpile/to-java.xsl

```diff
@@ -139,12 +139,12 @@
       <xsl:when test="$first='Q'">
         <xsl:text>Phi.Φ</xsl:text>
       </xsl:when>
-      <xsl:when test="$first='$'">
+      <xsl:when test="$first='ξ'">
         <xsl:value-of select="$rho"/>
       </xsl:when>
       <xsl:otherwise>
         <xsl:message terminate="yes">
-          <xsl:text>FQN must start with either with Q or $, but </xsl:text>
+          <xsl:text>FQN must start with either with Q or ξ, but </xsl:text>
```

Comment:
@h1alexbel maybe let's also get rid of `Q` and `QQ` in XMIR and replace them with `Φ` and `Φ̇`?
@yegor256 WDYT?